source/scpd/Cpp.d

```diff
@@ -111,6 +111,16 @@ extern(C++, `std`) {
     {
         void*[3] ptr;
     }
+
+    /// Simple bindings to std::chrono
+    public struct chrono
+    {
+        public static struct duration
```

Comment:
Isn't it a template class ?

Reply:
Hmm.. So I looked into the sources. To make `duration` templated I need to turn the struct into:

```D
public static struct duration (_Rep, _Period = ratio!1)
```

And then also bind to `ratio`, which is another templated struct:

```C++
namespace std _GLIBCXX_VISIBILITY(default)
{
    template<intmax_t _Num, intmax_t _Den = 1>
        struct ratio
        {
        }
}
```

Is it ok to bind to the `_GLIBCXX_VISIBILITY` namespace? Because it's specific to glibc and our D binding code might break on other platforms..

Reply:
The namespace is `std`, `_GLIBCXX_VISIBILITY` is just a macro (which does nothing in this case).